person.rb

```diff
@@ -1,23 +1,56 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
 class Person
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md) for this project, " `Make sure that your Person class inherits from Nameable
` ", please kindly fix that.